experiments/kompanion/README.md

```diff
@@ -26,7 +26,7 @@ Export function may take minutes to run as the tool searches all the api resourc
 	kompanion export --include-namespaces=my-team
 
 	# target only specific namespace prefixes AND specific object prefixes
-	kompanion export --target-namespaces=my-team --target-objects=important
+	kompanion export --target-namespaces=my-team --target-objects=logging
```

Comment:
Nit: From a UX perspective, this flag is somewhat obscure.  Maybe `--kind-prefixes=`.  I don't really know why we need "target"...